.travis.yml

```diff
@@ -22,9 +21,14 @@ install:
       - conda update -q conda
         # Useful for debugging any issues with conda
       - conda info -a
-      - conda install -q --yes --file requirements.txt -c cta-observatory -c conda-forge -c openastronomy
-      - python setup.py install
+      - conda create -n test python=3.6
+      - source activate test
```

Comment:
Recent conda versions use a different syntax for managing and activating environments.

Replace 
`export PATH="$HOME/miniconda/bin:$PATH"`

with 

```
. $HOME/miniconda/etc/profile.d/conda.sh
```

and 

```
source activate test
```

with
```
conda activate test
```